### ISSUE TITLE
Cannot read property 'appendChild' of null

### ISSUE THREAD

**Issue by btopro** [PolymerElements/iron-a11y-announcer#32]
I've done a build w/ iron-a11y-announcer 2.0.0 and while it doesn't impact the app the console will throw the error when attempting to append itself to the body of the document. Not sure how it's possible or if there needs to be some checking for body being ready before attempting to append.

![2017-08-05_10-29-27](https://user-images.githubusercontent.com/329735/28996196-268e3dfa-79c9-11e7-82b0-7ac5ff65c8b1.png)

**Comment by btopro** [comment]
Been doing some digging. This can happen because of elements like `iron-input` who invoke `Polymer.IronA11yAnnouncer.requestAvailability();` during their `created` callback. Seems like a race condition, potentially in larger DOMs where `document.body` doesn't exist yet.

**Comment by btopro** [comment]
I don't believe this is the best solution by any means but this resolves the errors from popping up
```
Polymer.IronA11yAnnouncer.requestAvailability = function() {
        document.addEventListener("DOMContentLoaded", function(event) {
          if (!Polymer.IronA11yAnnouncer.instance) {
            Polymer.IronA11yAnnouncer.instance = document.createElement('iron-a11y-announcer');
          }

          document.body.appendChild(Polymer.IronA11yAnnouncer.instance);
        });
      };
```

**Comment by ggriffithsIDBS** [comment]
Im also seeing this issue where document.body is null inside of the requestAvailability and .created callback

**Comment by btopro** [comment]
it doesn't hurt anything (or seem to) since things will check again for it being instantiated but still annoying and the above at least resolves it; I don't think it's the best way to go about it but it works
